.github/workflows/dev-cluster-check.yml

```diff
@@ -0,0 +1,40 @@
+name: Check development cluster
+on:
+  schedule:
+    - cron: '* 0/2 * * *'
+jobs:
+  run-integration-tests:
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Free disk space
+        uses: ./.github/actions/free-disk-space
+      - name: Add mask
+        run: |
+          echo "::add-mask::${{ secrets.DEV_CLUSTER_MANAGER_URL }}"
+          echo "::add-mask::${{ secrets.DEV_CLUSTER_KEY_CLOAK_URL }}"
+          echo "::add-mask::${{ secrets.DEV_CLUSTER_BRIDGE_TOKEN_USERNAME }}"
+          echo "::add-mask::${{ secrets.DEV_CLUSTER_BRIDGE_TOKEN_PASSWORD }}"
+      - name: Java and Maven Setup
+        uses: ./.github/actions/java-maven-setup
+        with:
+          cache-key-prefix: ${{ runner.os }}
+      - name: Compile sandbox repo
+        uses: ./.github/actions/maven
+        with:
+          maven-command: clean install -DskipTests
+      - name: Run integration tests
+        uses: ./.github/actions/maven
+        with:
+          maven-command: clean verify -Pcucumber -Devent-bridge.manager.url=${{ secrets.DEV_CLUSTER_MANAGER_URL }} -Dkey-cloak.url=${{ secrets.DEV_CLUSTER_KEY_CLOAK_URL }} -Dbridge.token.username=${{ secrets.DEV_CLUSTER_BRIDGE_TOKEN_USERNAME }} -Dbridge.token.password=${{ secrets.DEV_CLUSTER_BRIDGE_TOKEN_PASSWORD }}
```

Comment:
are the tests going to cleanup all the resources from the cluster with some `delete` operations?

Reply:
Cleanup is part of test teardown - https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/steps/End2EndTestITStep.java#L256-L265